src/ArcGISEarth.AutoAPI.Utils/AutomationAPIHelper.cs

```diff
@@ -23,19 +22,20 @@ namespace ArcGISEarth.AutoAPI.Utils
 {
     public class AutomationAPIHelper
     {
-        private const string TARGET_OPERATIONALLAYERS = "operationalLayers";
+        public const string CameraControllerName = "camera";
+        public const string FlightControllerName = "flight";
+        public const string LayerControllerName = "layer";
+        public const string LayersControllerName = "layers";
+        public const string WorkspaceControllerName = "workspace";
+        public const string SnapshotControllerName = "snapshot";
```

Comment:
`public`  ->  `private`

Reply:
use `public` so other class can use this. the same logic as the `APIBaseUrl` @gis2all

Reply:
use public so other class can use this. the same logic as the public `get` for  `APIBaseUrl` @gis2all

Reply:
already change `public` to `priivate` and merged this branch